app/controllers/grandmothers_controller.rb

```diff
@@ -0,0 +1,22 @@
+class GrandmothersController < ApplicationController
+  def new
+    @grandmother = Grandmother.new
+  end
+
+  def create
+    @grandmother = Grandmother.new(grandmother_params)
+    @grandmother.user = current_user
+
+    if @grandmother.save
+      redirect_to profile_path
```

Comment:
Sorry, maybe we need a notice to say the grandmother instance was created, justo give some feedback to the user